src/bringauto/fleet_protocol/http_client/RequestFrequencyGuard.cpp

```diff
@@ -0,0 +1,54 @@
+#include <bringauto/fleet_protocol/http_client/RequestFrequencyGuard.hpp>
+
+#include <unistd.h>
+#include <chrono>
+#include <thread>
+
+
+RequestFrequencyGuard::RequestFrequencyGuard(int32_t maxRequestsThresholdCount, int32_t maxRequestsThresholdPeriodMs,
+                                             int32_t delayAfterThresholdReachedMs, int32_t retryRequestsDelayMs) {
+    this->maxRequestsThresholdCount = maxRequestsThresholdCount;
+    this->maxRequestsThresholdPeriodMs = maxRequestsThresholdPeriodMs;
+    this->delayAfterThresholdReachedMs = delayAfterThresholdReachedMs;
+    this->retryRequestsDelayMs = retryRequestsDelayMs;
```

Comment:
change names, use func(from):to_(from){} notation

Reply:
changed private names and notation